common/.gitignore

```diff
@@ -0,0 +1 @@
+protobuf
```

Comment:
needed?
I guess we are going to remove it later

Reply:
given is currently necessary with instruction to create this from nanopb dist _and_ we do not want it commit i think it should be in .gitignore (if every this protobuf directory will become obsolete we obsolete should clean up also the .gitignore ..)

Reply:
from a refactoring perspective it seems to me to make sense to have local .gitignore if it is about particular files